tests/db/jobs/test_daemon.py

```diff
@@ -3,31 +3,44 @@
 
 import pytest
 
-# pytestmark = pytest.mark.skipif(
-#     not os.environ.get("CI"),
-#     reason="Only run integration tests in CI, unless forced with 'CI' env var",
-# )
 
-
-def _wait_daemon_started(daemon_manager, max_wait: int = 10) -> bool:
-    from jobflow_remote.jobs.daemon import DaemonStatus
+def _wait_daemon_shutdown(daemon_manager, max_wait: int = 10) -> bool:
+    from jobflow_remote.jobs.daemon import DaemonError, DaemonStatus
 
     for _i in range(max_wait):
         time.sleep(1)
-        state = daemon_manager.check_status()
-        assert state in (DaemonStatus.STARTING, DaemonStatus.RUNNING)
-        if state == DaemonStatus.RUNNING:
+        state = None
+        try:
+            state = daemon_manager.check_status()
+        except DaemonError:
+            pass
```

Comment:
Here you pass while in the similar fixture, you continue. Is there a reason ?
Also, is this function needed (see below comment as to why not using the fixture instead) ?

Reply:
I switched to the fixture. In the other case I use a continue because it checks that current state belongs to a list